src/DynamicSelect.php

```diff
@@ -20,17 +20,33 @@ public function resolve($resource, $attribute = null)
         return parent::resolve($resource, $attribute);
     }
 
+    public function placheholder($placeholder = 'Pick a value')
```

Comment:
These methods are not executed by default so we don't have a fallback anymore right? This way a user is forced to always specify these. Plus; they are not translatable by default anymore which makes it a breaking change.